scripts/update_all_products.pl

```diff
@@ -402,6 +421,55 @@
 			}
 		}
 
+		# Prefix untaxonomized packaging tags values with the most likely language
+		if ($prefix_packaging_tags_with_language and (defined $product_ref->{packaging}) and ($product_ref->{packaging} ne '')) {
+			my $current_packaging = $product_ref->{packaging};
+			my @new_tags = ();
+			foreach my $tag (split(/,/, $current_packaging)) {
+				$tag =~ s/\s+$//g;
+				$tag =~ s/^\s+//g;
+				my $new_tag = $tag;
+				# skip tags that are already prefixed with a a language
+				if ($tag =~ /^[a-z]{2}:/) {
+					$prefix_packaging_already_prefixed++;
+				}
+				else {
+					# first try the language of the product, then languages that are the most represented on OFF
+					my @potential_languages = ($product_ref->{lc}, "fr", "en", "de", "es", "it", "nl", "pt", "pl", "se", "ar", "cz", "ro", "bg", "nb", "da", "ru", "hu", "ca" );
+					my $l = guess_language_of_packaging_text($tag, \@potential_languages);
+					my $properties = "unrecognized_properties";
```

Comment:
I find it more readable to put the assignation in the else branch. But it's just an opinion.